src/main/java/cd/go/contrib/elasticagents/docker/Constants.java

```diff
@@ -28,7 +28,7 @@
     String EXTENSION_TYPE = "elastic-agent";
 
     // The extension point API version that this plugin understands
-    String API_VERSION = "2.0";
+    String API_VERSION = "3.0";
```

Comment:
Can we use a different version constant for requests from server and request from the plugin?

Plugin request processor version is still 1.0 this was a bug in the skeleton plugin. 

https://github.com/gocd-contrib/docker-elastic-agents/blob/90de4926d4a878626c1e241d2dd775adb95048b8/src/main/java/cd/go/contrib/elasticagents/docker/PluginRequest.java#L40

https://github.com/gocd-contrib/docker-elastic-agents/blob/90de4926d4a878626c1e241d2dd775adb95048b8/src/main/java/cd/go/contrib/elasticagents/docker/PluginRequest.java#L51

https://github.com/gocd-contrib/docker-elastic-agents/blob/90de4926d4a878626c1e241d2dd775adb95048b8/src/main/java/cd/go/contrib/elasticagents/docker/PluginRequest.java#L66

https://github.com/gocd-contrib/docker-elastic-agents/blob/90de4926d4a878626c1e241d2dd775adb95048b8/src/main/java/cd/go/contrib/elasticagents/docker/PluginRequest.java#L81